lib/highscore.input.lua

```diff
@@ -1,30 +1,19 @@
 _hs_name="aaa"
 _hs_name_index=1
-_hs_name_table={1,1,1}
+_hs_name_table={0,0,0}
```

Comment:
Oops. Wrong index.

Reply:
The code was working correctly before my change (and should be working correctly afterwards) -- I just changed these indexes so that I could use the `%` operator more easily in the input section. (specifically, I wanted to write `foo%num` instead of `(foo-1)%num + 1`)